tests/it/it_scargo_commands_flow.py

```diff
@@ -1,12 +1,19 @@
+import json
 import os
+from dataclasses import dataclass
+from enum import Enum
 from pathlib import Path
-from shutil import copy, copytree
+from shutil import copytree
+from typing import List, Optional
 from unittest.mock import patch
```

Comment:
1. I would add pytest.order mark to all tests, so it would look more like test suite, right now some of the tests looks more like unit tests and it's not clear that they depends on previous results
2. At the begining of the file I would add comment to github issue with test plan for those tests